packages/backend/src/core/events/findCorrespondingBlocks.ts

```diff
@@ -0,0 +1,48 @@
+import { UnixTime } from '@l2beat/types'
+import { sortBy } from 'lodash'
+
+interface BlockInfo {
+  timestamp: UnixTime
+  blockNumber: bigint
+}
+
+export function findCorrespondingBlocks<T extends { blockNumber: number }>(
+  _blocks: BlockInfo[],
+  _logs: T[],
```

Comment:
Why are argument variables prefixed with `_`? We don't use this convention anywhere else